.github/workflows/ci.yml

```diff
@@ -659,13 +660,13 @@ jobs:
       - name: Memsql Tests
         env:
           MEMSQL_LICENSE: ${{ secrets.MEMSQL_LICENSE }}
-        if: matrix.modules == 'plugin/trino-singlestore' && env.MEMSQL_LICENSE != ''
+        if: matrix.modules == 'plugin/trino-singlestore' && (env.CI_SKIP_SECRETS_PRESENCE_CHECKS != '' || env.MEMSQL_LICENSE != '')
```

Comment:
does this mean this cannot access secrets.CI_SKIP_SECRETS_PRESENCE_CHECKS directly?

Reply:
Right. It's mentioned in the official documentation. 
> Secrets cannot be directly referenced in if: conditionals
https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow